src/main/scala/fi/aalto/cs/apluscourses/intellij/utils/ModuleUtils.scala

```diff
@@ -34,4 +36,66 @@ object ModuleUtils {
     nonEmpty
   }
 
+  def naiveValidate(@NotNull command: String): Boolean =
+    command.matches("import\\so1\\.[a-z]*(\\_|\\._)$")
+
+  def clearCommands(@NotNull imports: Array[String]): Array[String] =
+    imports
+      .clone
+      .map(_.replace("import ", ""))
+      .map(_.replace("._", ""))
+
+  def getCommandsText(@NotNull imports: Array[String]): String =
+    imports.length match {
+      case 0 => ""
+      case 1 => "Auto-imported package [" + imports(0) + "] for your convenience." // scalastyle:ignore
+      case _ => "Auto-imported packages [" + imports.mkString(", ") + "] for your convenience."
+    }
```

Comment:
Nice! Two nits:

I think replacing `imports(0)` with `imports.head` would get rid of the scalastyle warning.

Also a possibility to use string interpolation
`s"Auto-imported package [${imports.head}] for your convenience."`